src/svg/AuFlag.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
svgr does have a Rollup plugin, but it would require to implement a custom Rollup config.  In my opinion the benefit does not justify the effort and the future cost of maintaining a custom Rollup config